csharp/protos/extensions/extensions_b.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+import "extensions_a.proto";
+
+option csharp_namespace = "Google.Protobuf.TestProtos.Extensions";
+
+// This file is used as part of a unit test for issue 6936
+// We don't need to use it, we just have to import it in c
+
+message Foo {
+    option (opt) = "bar";
```

Comment:
qq: why does Foo have option value "bar" and vice versa?

Reply:
Just thought it up on a whim. I can change it if you think it doesn't work.